### PR TITLE
keep default resolv search list in mantl-dns conf

### LINKED ISSUE #84: mantl-dns: dnsmasq search configuration
see https://github.com/CiscoCloud/mantl/issues/1367 and https://github.com/CiscoCloud/mantl/issues/1377. DNS was not able to resolve aws internal host names. Using the `--domain='#'` option and a single resolv file [1] addressed the problem.

[1] we manually tested by running this:

`cat /etc/resolv.conf.mantl-dns /etc/resolv.conf.mas1 > /etc/resolv.conf`

### PR DESCRIPTION
**Pull request by ryane**
fixes #84

### CODE REVIEW

File: mantl/mantl-dns/spec.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name: mantl-dns
 version: 1.1.0
 license: ASL 2.0
-iteration: 2
+iteration: 3
```
Comment (ryane) [inline mantl/mantl-dns/spec.yml]: is bumping the iteration sufficient or should the version change?

File: mantl/mantl-dns/spec.yml [hunk 2]
```diff
@@ -54,6 +58,11 @@ scripts:
         ln -s /etc/resolv.conf.mantl-dns /etc/resolv.conf
     fi
 
+  after-upgrade: |
+    # update the installed config search list with resolv.conf values
+    search=$(grep ^search /etc/resolv.conf.masq | sed -e "s/^search\s//" -e "s/consul\s//")
+    sed -i -e "s/^search.*$/search node.consul consul $search/" /etc/resolv.conf.mantl-dns
+
```
Comment (ryane) [inline mantl/mantl-dns/spec.yml]: on upgrade, the whole line is replaced to avoid duplicates

**Comment by ryane** [comment]
@BrianHicks @ChrisAubuchon can you take a look and see if this looks ok to you?

**Comment by BrianHicks** [comment]
Looks fine but I can't test ATM

**Comment by ryane** [comment]
thanks

**Pull request merged**